pkg/runner/expression.go

```diff
@@ -47,6 +48,7 @@ func (rc *RunContext) NewExpressionEvaluator(ctx context.Context) ExpressionEval
 		Steps: rc.getStepsContext(),
 		Runner: map[string]interface{}{
 			"os":         "Linux",
+			"arch":       container.RunnerArch(ctx),
 			"temp":       "/tmp",
 			"tool_cache": "/opt/hostedtoolcache",
 		},
```

Comment:
> I'm not shure if we need to use docker inspect on the jobcontainer to get reliable the arch of the container, but this can wait until the HostExecution Environment is merged and we have full access to the container structure.

Okay cool - thanks for getting back. 
I am fine with using my fork for now 🙂